image-layout.md

```diff
@@ -21,7 +21,7 @@ It also contains a file that is used to identify the layout version:
 
 This is an example image layout:
 
-```
+```shell
```

Comment:
This language specification if for the shell language, but the following example is a *shell session*.  It has both shell-language input (which the `shell` you add formats correctly) and unformatted output (which the `shell` you add *also* attempts to format).  As an example:

```shell
$ echo "in this case, these are not shell keywords"
in this case, these are not shell keywords
```

I'd rather leave these as language-agnostic blobs to avoid confusion.